github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml

```diff
@@ -16,34 +16,69 @@ periodics:
     - org: kubevirt
       repo: kubevirtci
       base_ref: master
-      work_dir: true
     - org: kubevirt
       repo: common-templates
-      base_ref: master
+      base_ref: containerdisk-automation
+      work_dir: true
   spec:
     nodeSelector:
       type: bare-metal-external
     containers:
-    - image: quay.io/kubevirtci/bootstrap:v20210126-a12b6c0
+    - image: quay.io/kubevirtci/kubevirt-infra-bootstrap:v20210331-b66b42e
       command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
         - "-c"
         - |
-          apt-get update && apt-get install -y --no-install-recommends --no-upgrade expect ansible intltool libosinfo-1.0 libssl-dev osinfo-db-tools python-gi && rm -rf /var/lib/apt/lists/*
-          export KUBEVIRTCI_TAG=$(curl -L https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)
-          export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:${KUBEVIRTCI_TAG}
-          export KUBEVIRT_PROVIDER=k8s-1.20
           export KUBEVIRT_NUM_NODES=3
```

Comment:
This env var should be defined on a `env` property, like here https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml#L261-L263

Reply:
Moved it into the script